tests/Database/DatabaseEloquentBuilderTest.php

```diff
@@ -511,6 +546,15 @@ public function testDeleteOverride()
         $this->assertEquals(['foo' => $builder], $builder->delete());
     }
 
+    public function testAggregatedValuesOfDatetimeField()
+    {
+        EloquentBuilderTestUser::create(['id' => 1, 'email' => 'test1@test.test', 'created_at' => '2016-08-10 09:21:00']);
+        EloquentBuilderTestUser::create(['id' => 2, 'email' => 'test2@test.test', 'created_at' => '2016-08-01 12:00:00']);
+
+        $this->assertEquals('2016-08-10 09:21:00', EloquentBuilderTestUser::max('created_at'));
```

Comment:
Also I think the phpdoc said that it could return a string or mixed before. The behavior has not changed.

If you are at larecon I won't mind discussing it with you